lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -130,6 +131,10 @@ abstract class AssetPickerBuilderDelegate<Asset, Path> {
   final AssetsChangeRefreshPredicate<AssetPathEntity>?
       assetsChangeRefreshPredicate;
 
+  /// List of prepend special items.
+  /// 前置自定义item列表
+  final List<SpecialItem<Path>> prependSpecialItems;
```

Comment:
Why do we need prepend items and why don't we need append items?

Reply:
`prependSpecialItems` is a reusable variable to get prepend item and it is used in `findChildIndexBuilder` and `semanticIndex` function. At the moment, there is no append items needed in anywhere.
![image](https://github.com/user-attachments/assets/8f2550ce-84d2-4540-91d8-305fc3535fb1)
![image](https://github.com/user-attachments/assets/b4b31d29-174e-4a3b-abd2-56753db80726)
![image](https://github.com/user-attachments/assets/4aecc6ad-fb35-4512-8cba-69b291bdb5b1)

Reply:
It might be better to keep interfaces consist since the delegate is also exposed to users.

Reply:
done added `appendSpecialItems`.